src/components/List/List.tsx

```diff
@@ -33,6 +33,12 @@ export interface ListProps extends UIComponentProps, ChildrenComponentProps {
   /** A selection list formats list items as possible choices. */
   selection?: boolean
 
+  /** Index of the currently selected item. */
+  selectedItemIndex?: number
```

Comment:
just `selectedIndex`?

Reply:
then ```focusedIndex``` to keep consistency. I am ok with both variants